docs/how-tos/upload-data.md

```diff
@@ -19,7 +19,7 @@ aliases:
   - /services/data/upload/
 ---
 
-If you configured [data capture](/services/data/capture/) on your machine, Viam automatically uploads data to the cloud from the directory you specified, at the interval you specified.
+If you configured [data capture](/services/data/capture-sync/) on your machine, Viam automatically uploads data to the cloud from the directory you specified, at the interval you specified.
```

Comment:
small copy editing suggestion, feel free to take it or leave it :)

"If you configured [data capture](https://github.com/services/data/capture-sync/) on your machine, Viam automatically uploads data to the cloud from the directory you specified, at the interval you specified."

"If you configured [data capture](https://github.com/services/data/capture-sync/), Viam automatically uploads data from the specified directory to the cloud at your designated interval."

Reply:
```suggestion
If you configured [cloud sync](/services/data/capture-sync/), Viam automatically uploads data from the specified directory to the cloud, at the interval you specified.
```